templates/ts-input/index.ts

```diff
@@ -0,0 +1,12 @@
+import { ReadableApp  } from "@scramjet/types";
```

Comment:
All ts templates should be the same code as js templates, but with added typescript types, otherwise it will confuse user.